module1/src/com/epam/cycles/HomeTask_1_3_5.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.cycles;
+
+public class HomeTask_1_3_5 {
+    public static void main(String[] args) {
+        for (int i = 0; i <= 127; i++) {
+            System.out.println(i + " " + (char) i);
+        }
```

Comment:
1)move to constant, let's say private static final int  ASCII_LENGTH = 255;
2)you can go until 255
```suggestion
    private static final int  ASCII_LENGTH = 255;
    
    public static void main(String[] args) {
        for (int i = 0; i <= ASCII_LENGTH; i++) {
            System.out.println(i + " " + (char) i);
        }
```